mapproxy/config/loader.py

```diff
@@ -1019,6 +1020,31 @@ def source(self, params=None):
 }
 
 
+def register_source_configuration(config_name, config_class,
```

Comment:
I think this method and also e.g. `register_service_configuration` should use `log` to produce meaningful output in the logfile, so that loaded plugins can be easily detected from the logfile.

Reply:
done